compiler/damlc/tests/daml-test-files/MaybeCompat.daml

```diff
@@ -1,11 +1,13 @@
 -- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @WARN range=14:12-14:17; Maybe
--- @WARN range=17:12-17:17; maybe
--- @WARN range=17:29-17:36; Nothing
--- @WARN range=18:12-18:20; fromSome
--- @WARN range=18:22-18:26; Just
+-- @INFO Use fromOptional
+-- @WARN Maybe
```

Comment:
Please don’t remove ranges that we already have. That makes it easy to introduce regressions in our location info.

Reply:
Restored